projects/picker/src/lib/date-time/date-time-inline.component.ts

```diff
@@ -311,6 +322,19 @@ export class OwlDateTimeInlineComponent<T> extends OwlDateTime<T> implements OnI
     } else {
       this.value = date;
     }
+
```

Comment:
Also maybe add a comment here about what your new logic does. The whole component is already complex enough 😅